### PR TITLE
feat: add platform cli docs

### PR DESCRIPTION
**Pull request by ankur-arch**
PDP CLI docs based on this [doc](https://www.notion.so/prismaio/Platform-CLI-ver-Feb-20-Manual-QA-83571e2847024c1d81c191e8b4e1e55d) in Early Access.

### CODE REVIEW

File: content/500-platform/60-platform-cli/10-about.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: 'About'
+metaTitle: 'Platform CLI: About'
+tocDepth: 3
+---
+
+<TopBlock>
+
+This documentation provides details on how to access the Prisma Data Platform using the Prisma CLI. Follow the [installation guide](/orm/tools/prisma-cli#installation) for the Prisma CLI to start using it.
```
Comment (luanvdw) [inline content/500-platform/60-platform-cli/10-about.mdx]:
```suggestion
This guide demonstrates how to access the Prisma Data Platform using the [Prisma CLI](/orm/tools/prisma-cli). Get started by ensuring you have the Prisma CLI installed, following our setup [instructions](/orm/tools/prisma-cli#installation).
```

File: content/500-platform/60-platform-cli/10-about.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+The integration, currently in [Early Access](/platform/maturity-levels#early-access), extends the functionality of the Prisma CLI, enabling you to manage your Platform projects. For the complete list of available commands, navigate to the [Commands page](/platform/platform-cli/commands).
```
Comment (luanvdw) [inline content/500-platform/60-platform-cli/10-about.mdx]:
```suggestion
This integration, now in Early Access, expands the capabilities of the Prisma CLI, enabling management of your Platform projects directly from the command line. For a detailed overview of all the commands, please refer to the [Commands page](/platform/platform-cli/commands).
```

File: content/400-pulse/600-faq.mdx [hunk 3]
```diff
@@ -44,5 +44,26 @@ Pulse’s database compatibility list will be different from Prisma Client's dat
...
 | Railway  | ✅       | Requires [`sslmode=disable`](/orm/overview/databases/postgresql#configuring-an-ssl-connection) due to the use of a self-signed certificate for database connections by Railway.  |
 | Supabase | ✅       | Requires [`sslmode=disable`](/orm/overview/databases/postgresql#configuring-an-ssl-connection) due to the use of a self-signed certificate for database connections by Supabase. |
-| Neon     | _Coming_ | Support details coming soon.                                                                                                 |
+| Neon     | _Coming_ | Support details coming soon.                                                                                                                                                     |
 | AWS RDS  | _Coming_ | Support details coming soon.                                                                                                                                                     |
+
+## Which regions is Prisma Pulse available in?
+
+Currently, the list of available regions are:
```
Comment (luanvdw) [inline content/400-pulse/600-faq.mdx]: @ankur-arch this is incorrect, it's a bit nuanced now with the AWS routing for self-signed certificates. I'll get the final list from eng.

File: content/500-platform/20-concepts/30-environments.mdx [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+<TopBlock>
+
+Environments offer isolated spaces for teams to configure, test, and refine projects across different stages, from experimentation to production. This setup ensures focused development and seamless progress, free from interference.
+
+</TopBlock>
+
+## API Keys
+
+An API key is required to authenticate requests from your Prisma Client to products such as Accelerate and Pulse. You may generate multiple API keys per environment and manage those via the **Settings** tab of a project's page.
```
Comment (luanvdw) [inline content/500-platform/20-concepts/30-environments.mdx]:
```suggestion
An API key is required to authenticate requests from your Prisma Client to products such as Accelerate and Pulse. You may generate multiple API keys per environment and manage those via the **API Keys**.
```

File: content/200-orm/500-reference/200-prisma-cli-reference.mdx [hunk 5]
```diff
@@ -1342,6 +1342,52 @@ Other options:
    --script > script.sql
   ```
 
+## Prisma Data Platform
+
+### <inlinecode>platform</inlinecode> ([Early Access](/platform/maturity-levels#early-access))
+
+The `platform` command provides access to the Prisma Data Platform through the Prisma CLI starting in version `5.7.0` or later.
```
Comment (luanvdw) [inline content/200-orm/500-reference/200-prisma-cli-reference.mdx]:
```suggestion
The `platform` command provides access to the Prisma Data Platform through the Prisma CLI starting in version `5.10.0` or later.
```

Where highlighted, we should advocate for the latest version of Prisma to utilize the Platform integration in the CLI

**Review by luanvdw — COMMENTED** [review]
@ankur-arch thank you for putting this together! I added a few comments below

File: content/400-pulse/600-faq.mdx [hunk 6]
```diff
@@ -44,5 +44,26 @@ Pulse’s database compatibility list will be different from Prisma Client's dat
...
 | Railway  | ✅       | Requires [`sslmode=disable`](/orm/overview/databases/postgresql#configuring-an-ssl-connection) due to the use of a self-signed certificate for database connections by Railway.  |
 | Supabase | ✅       | Requires [`sslmode=disable`](/orm/overview/databases/postgresql#configuring-an-ssl-connection) due to the use of a self-signed certificate for database connections by Supabase. |
-| Neon     | _Coming_ | Support details coming soon.                                                                                                 |
+| Neon     | _Coming_ | Support details coming soon.                                                                                                                                                     |
 | AWS RDS  | _Coming_ | Support details coming soon.                                                                                                                                                     |
+
+## Which regions is Prisma Pulse available in?
```
Comment (janpio) [inline content/400-pulse/600-faq.mdx]: Offtopic for this PR?

Comment (ankur-arch) [reply content/400-pulse/600-faq.mdx]: @janpio, it is off-topic, but I included it as I'm referring to this in the CLI command to enable Prisma Pulse. 
Particularly for the `region` flag during creation.

**Review by jasonkuhrt — APPROVED** [review]
Re CLI stuff, LGTM!

**Review by luanvdw — APPROVED** [review]
@ankur-arch LGTM! :)

**Pull request merged**